mender-deb-package

```diff
@@ -149,12 +154,18 @@ build_packages() {
     sign_flags="--sign-key=$key_id"
   fi
 
+  # dh_make needs a name of the user in the env
+  export USER=root
+  # needed to get the orig file
+  echo dh_make --createorig --packagename "${DEB_PACKAGE}_${DEB_VERSION_NO_REV}" -s -y || true
+  dh_make --createorig --packagename "${DEB_PACKAGE}_${DEB_VERSION_NO_REV}" -s -y || true
```

Comment:
Why the `|| true` ? Shouldn't we make the script fail if the orig cannot be created?

Reply:
the problem here is: dh_make will exit with 1 because debian directory exists -- but do not worry, if the orig file is not created here, the next command fails -- so no it should be ok.

Reply:
ok